src/App.scss

```diff
@@ -0,0 +1,78 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
```

Comment:
this needs to be moved out into clock.scss then imported into this file

clock.scss should be in the clock folder i think